### ISSUE TITLE
Can't use redux-storage in react-boilerplate anymore

### ISSUE THREAD

**Issue by Proxiweb** [michaelcontento/redux-storage#152]
I used to add redux-storage to the react-boilerplate without any problems. I've updated both projects and there is an error... I can't figure what's going on... When I configure my store as ususal I get 
```
ERROR in ./~/redux-storage/src/index.js
Module parse failed: /home/regisg/projets/foodtest/node_modules/redux-storage/src/index.js Unexpected token (8:4)
You may need an appropriate loader to handle this file type.
| // The full default export is required to be BC with redux-storage <= v1.3.2
| export default {
|     ...require('./constants'),
|     createLoader: require('./createLoader').default,
|     createMiddleware: require('./createMiddleware').default,
 @ ./app/store.js 11:0-41
```
I've tried many presets/plugins settings in both projects without success...

**Comment by fforres** [comment]
Having the same problem. @Proxiweb  Found a solution yet? 
I believe it's something regarding babel versions.

Would you mind copying your package.json's dependencies and devDependencies ?

**Comment by Proxiweb** [comment]
I haven't found a solution. I dropped redux-storage and implemented a simpler solution to just store part of the state in sessionStorage.

```
  "dependencies": {
    "babel-polyfill": "6.13.0",
    "bcrypt-nodejs": "0.0.3",
    "body-parser": "^1.15.2",
    "bootstrap-css-only": "^3.3.6",
    "chalk": "1.1.3",
    "compression": "1.6.2",
    "express": "4.14.0",
    "express-jwt": "^3.4.0",
    "fontfaceobserver": "2.0.1",
    "getconfig": "^3.0.0",
    "history": "3.0.0",
    "immutable": "3.8.1",
    "intl": "1.2.4",
    "invariant": "^2.2.1",
    "ip": "1.1.3",
    "jsonwebtoken": "^7.1.8",
    "lodash": "^4.14.1",
    "moment": "^2.14.1",
    "react": "15.3.0",
    "react-addons-update": "^15.3.0",
    "react-bootstrap": "^0.30.2",
    "react-dom": "15.3.0",
    "react-google-login": "^2.1.2",
    "react-helmet": "3.1.0",
    "react-intl": "^2.1.3",
    "react-redux": "4.4.5",
    "react-router": "2.6.1",
    "react-router-bootstrap": "^0.23.1",
    "react-router-redux": "4.0.5",
    "react-router-scroll": "0.2.1",
    "redux": "3.5.2",
    "redux-immutable": "3.0.6",
    "redux-saga": "0.11.0",
    "redux-storage": "^4.1.0",
    "redux-storage-engine-sessionstorage": "^1.0.2",
    "reselect": "2.5.3",
    "sanitize.css": "4.1.0",
    "superagent": "^2.1.0",
    "thinky": "^2.3.4",
    "warning": "^3.0.0"
  },
  "devDependencies": {
    "babel-cli": "^6.11.4",
    "babel-core": "6.13.2",
    "babel-eslint": "6.1.2",
    "babel-loader": "6.2.4",
    "babel-plugin-react-intl": "^2.1.3",
    "babel-plugin-react-transform": "2.0.2",
    "babel-plugin-transform-react-constant-elements": "6.9.1",
    "babel-plugin-transform-react-inline-elements": "6.8.0",
    "babel-plugin-transform-react-remove-prop-types": "0.2.9",
    "babel-preset-es2015": "6.13.2",
    "babel-preset-react": "6.11.1",
    "babel-preset-react-hmre": "1.1.1",
    "babel-preset-stage-0": "6.5.0",
    "chai": "3.5.0",
    "chai-enzyme": "0.5.0",
    "cheerio": "0.20.0",
    "coveralls": "2.11.12",
    "cross-env": "2.0.0",
    "css-loader": "0.23.1",
    "enzyme": "2.4.1",
    "eslint": "3.2.2",
    "eslint-config-airbnb": "10.0.0",
    "eslint-import-resolver-webpack": "0.4.0",
    "eslint-plugin-import": "1.12.0",
    "eslint-plugin-jsx-a11y": "2.0.1",
    "eslint-plugin-react": "6.0.0",
    "eventsource-polyfill": "0.9.6",
    "expect": "1.20.2",
    "expect-jsx": "2.6.0",
    "exports-loader": "0.6.3",
    "extract-text-webpack-plugin": "1.0.1",
    "file-loader": "0.9.0",
    "html-loader": "0.4.3",
    "html-webpack-plugin": "2.22.0",
    "image-webpack-loader": "2.0.0",
    "imports-loader": "0.6.5",
    "isparta": "4.0.0",
    "isparta-loader": "2.0.0",
    "json-loader": "0.5.4",
    "karma": "1.1.2",
    "karma-chrome-launcher": "1.0.1",
    "karma-coverage": "1.1.1",
    "karma-firefox-launcher": "1.0.0",
    "karma-ie-launcher": "1.0.0",
    "karma-mocha": "1.1.1",
    "karma-mocha-reporter": "2.1.0",
    "karma-safari-launcher": "1.0.0",
    "karma-sourcemap-loader": "0.3.7",
    "karma-webpack": "1.7.0",
    "lint-staged": "2.0.3",
    "minimist": "1.2.0",
    "mocha": "3.0.1",
    "ngrok": "2.2.2",
    "null-loader": "0.1.1",
    "offline-plugin": "3.4.2",
    "plop": "1.5.0",
    "postcss-cssnext": "2.7.0",
    "postcss-focus": "1.0.0",
    "postcss-loader": "0.9.1",
    "postcss-reporter": "1.4.1",
    "pre-commit": "1.1.3",
    "psi": "2.0.4",
    "rimraf": "2.5.4",
    "shelljs": "0.7.3",
    "sinon": "2.0.0-pre",
    "style-loader": "0.13.1",
    "stylelint": "7.1.0",
    "stylelint-config-standard": "12.0.0",
    "url-loader": "0.5.7",
    "webpack": "2.1.0-beta.15",
    "webpack-dev-middleware": "1.6.1",
    "webpack-hot-middleware": "2.12.2"
  }
```

**Comment by michaelcontento** [comment]
Fixed with 978760e0abcbee1c78c2079677182dca3c92da31

And sorry for the late response!

**Comment by fforres** [comment]
Thanks a lot @michaelcontento
